codeland.html

```diff
@@ -767,3 +843,35 @@ <h2 class="codeland-section__title align-center">
     </div>
   </section>
 </div>
+
+<script>
```

Comment:
I don't mind overhauling this altogether with better patterns/approach, but for a POC it gets the job done ✨